x-pack/legacy/plugins/reporting/config.ts

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { BROWSER_TYPE } from './common/constants';
+// @ts-ignore untyped module
+import { config as appConfig } from './server/config/config';
+import { getDefaultChromiumSandboxDisabled } from './server/browsers';
+
+export async function config(Joi: any) {
```

Comment:
@joelgriffith and I have been wanting to split up the Reporting index file for awhile. After converting it to Typescript and having all this code go through prettier, it made sense to break out this config section from the Reporting index.

Reply:
+1000